part-spec/power.json

```diff
@@ -1222,6 +1222,26 @@
           "$ref": "definitions.json#/unit"
         }
       }
+    },
+    "instance": {
+      "type": "object",
+      "properties": {
+        "componentTitle": {
+          "description": "title used for the component in the digital datasheets specifications",
+          "type": "string"
+        },
+        "instanceName": {
+          "description": "name of component instances",
+          "type": "string"
+        },
+        "instanceProperties": {
+          "description": "definition of the component instance as defined in the specifications",
+          "type": "array",
+          "items": {
+            "type": "TODO"
```

Comment:
We need to put something here, but I can't really tell how to refer to what this is.

From the [PMIC example](https://github.com/edatasheets/digital-datasheets/blob/main/support-docs/examples.md?plain=1#L1697) it looks like the object in the `instances` field is supposed to be whatever the top-level object of this file is (i.e. it seems like many power objects are valid here), but I'm not quite sure how to do that (or if it makes sense).

Any advice would be helpful!

Reply:
My proposal is to use object

Reply:
I think that would solve the recursive dependency, but it leaves a lot to be desired for this specification: we want to tell people what types of objects are valid here; we don't want people to have to deal with an infinite number of object types for this field.

Reply:
I agree with you Kevin. I don't have a better idea than Object though, but I agree that it's a bit weird.

Maybe it would make sense to keep this as Object and review this again for v0.9 for the purposes of getting 0.8 out?

Reply:
Good with me

Reply:
Done